data/remnant/remnant jobs.txt

```diff
@@ -223,6 +224,115 @@ mission "Remnant: Bounty 5"
 
 
 
+mission "Remnant: Logistics Civil Engineering"
+	name "Civil delivery to <planet>"
+	description `Engineers on <planet> need <cargo> delivered. Logistics adjustment is <payment>.`
+	cargo "remnant civil" 30 80
+	job
+	repeat
+	to offer
+		not "Remnant: Ssil Vida Active"
```

Comment:
These `to offer` and `source` changes apply to all the missions.
```suggestion
		not "remnant: ssil vida active"
```

Reply:
Corrected for all jobs.